test/test_any_automated_install.py

```diff
@@ -427,37 +434,13 @@ def test_FTL_detect_unknown_no_errors(host):
     assert expected_stdout in detectPlatform.stdout
 
 
-def test_FTL_download_aarch64_no_errors(host):
-    """
-    confirms only aarch64 package is downloaded for FTL engine
-    """
-    # mock dialog answers and ensure installer dependencies
-    mock_command("dialog", {"*": ("", "0")}, host)
-    host.run(
-        """
-    source /opt/pihole/basic-install.sh
-    package_manager_detect
-    install_dependent_packages ${INSTALLER_DEPS[@]}
-    """
-    )
-    download_binary = host.run(
-        """
-    source /opt/pihole/basic-install.sh
-    create_pihole_user
-    FTLinstall "pihole-FTL-aarch64-linux-gnu"
-    """
-    )
-    expected_stdout = tick_box + " Downloading and Installing FTL"
-    assert expected_stdout in download_binary.stdout
-    assert "error" not in download_binary.stdout.lower()
-
-
 def test_FTL_development_binary_installed_and_responsive_no_errors(host):
     """
     confirms FTL development binary is copied and functional in installed location
     """
     host.run(
         """
+    echo "development-v6" > /etc/pihole/ftlbranch
```

Comment:
Copy& Paste error. This is a duplicate of L464

Reply:
I'll remove 464 so they are all consistent in echo placement